ONSdigital/dp-filter-api
# Make mongodb updates granular

### What

- Add a timestamp to avoid race conditions on database updates
- Update unit tests
- Relies on https://github.com/ONSdigital/go-ns/pull/96

### How to review

- Verify logic is correct
- Run unit tests
- Run acceptance tests

### Who can review

Anyone
